File: ashritha_shetty/sum_range.cpp
/*
Created on Novemberer 08, 2018
@author: ashritha27
*/
//Program to find the sum of numbers in a given range in C++
#include <iostream>

using namespace std;

int main()
{
    //Declaring the range variables and initialising the sum variable
    int min,max,sum=0;
    
    cout<<"Enter the limits";
    
    //Read the minimum and maximum input
    cin>>min;
    cin>>max;
    
    //Summing all the numbers in the given range using for loop
    for(int i=min;i<=max;i++)
    {
         sum+=i;
    }
    
    //Prints the sum of numbers in the given range 
    cout<<sum;
    
    return 0;
}

File: sum_range.py
''' Program to find the sum of all numbers in the given range [A,B]
   Created on November 08, 2018
  	@Author : Apoorva kumar P
  	Input : A start variable A and an end variable B representing the range [A, B]
  	Output : The sum of all integers from A and B(both inclusive)
'''


#initializing the variable to store sum
sum=0;
#inputing start value  
start = int(input("Enter Start value:"))
#inputing end value
end   = int(input("Enter End value:"))
#for loop to determine the sum of all integers in the range [A,B]
for i in range(start,end+1):
   sum=sum + i
#printing sum value
print(sum)  
